### ISSUE TITLE
Be careful checking R version with binary packages

### ISSUE THREAD

**Issue by wch** [MangoTheCat/pkgsnap#21]
With `restore(needed_only=TRUE)`, pkgsnap compares the exact version of R to the version used to build each package. If a user is running R 3.3.1 and CRAN has binaries built on 3.3.2, then it will reinstall those packages built with 3.3.2, even though it's not really needed.

**Comment by gaborcsardi** [comment]
Yeah, for this you need to download the package, anyway, and then installing a binary package is really fast. So I am not sure if it is worth checking for this....
